src/app/components/formElements/Toggle/toggle.module.css

```diff
@@ -0,0 +1,86 @@
+.toggleRoot {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  position: relative;
+}
+
+.toggleContainer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.toggleContainer.vertical {
+  flex-direction: column;
+  gap: 5px;
+}
+
+.switchContainer {
+  display: flex;
+}
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 44px;
+  height: 24px;
+}
+
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--black-400);
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: var(--toggle-slider-radius);
+  width: var(--toggle-slider-radius);
+  top: 3px;
+  left: 3px;
+  background-color: var(--pure-white);
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+input:checked + .slider {
+  background-color: var(--cherryPieVariant);
+}
+
+input:focus + .slider {
+  box-shadow: 0 0 1px var(--cherryPieVariant);
+}
+
+input:checked + .slider:before {
```

Comment:
Browsers support single colon `:` before pseudo elements (like `before`, `after`) for backward compatibility, but the correct syntax require double colon. Replace all the instances of `:before` with `::before`.